data/jibauni/rebel jibauni 0 consultations.txt

```diff
@@ -0,0 +1,362 @@
+# Copyright (c) 2024 by Loymdayddaud
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+mission "First Contact: Rebel Jibauni"
+	name "Travel to <planet>"
+	description `Travel to <destination> to meet up with the leaders of this "Rebel Jibauni" faction.`
+	landing
+	source
+		government "Jibauni (Rebel)"
+	destination Andwerfen
+	to offer
+		not "Jibauni: Kanlanmo 0: active"
+		not "Jibauni: Kanlanmo 0: done"
+		not "Jibauni: Kanlanmo (0 declined): active"
+		not "Jibauni: Kanlanmo (0 declined): done"
+	on offer
+		conversation
+			branch "flagship andwerfen"
+				has "flagship planet: Andwerfen"
+			branch "met and declined the jibauni"
+				has "jibauni: joined the rebels"
+				not "flagship planet: Andwerfen"
+			branch "met and declined the jibauni and on andwerfen"
+				has "jibauni: joined the rebels"
+				not "flagship planet: Andwerfen"
+			label nothing
+			`As you land on <origin>, a strange creature comes up to you. It is in the shape of the fabled Tyrannosaurus Rex from Earth, but it is covered in what is seemingly metal armor.`
```

Comment:
After landing on <origin>, an unusual creature approaches you. It looks like a long extinct Tyrannosaurus Rex from Earth, but in miniature as it is no taller than yourself. It is also covered in a metallic looking armor.
```suggestion
			`As you land on <origin>, a strange creature comes up to you. It is in the shape of the fabled Tyrannosaurus Rex from Earth, but it is covered in what is seemingly metal armor.`
```